lib/Dancer2/Core/Role/Config.pm

```diff
@@ -281,8 +255,46 @@ __END__
 
 =head1 DESCRIPTION
 
-Provides a C<config> attribute that feeds itself by finding and parsing
-configuration files.
+This is the redesigned C<Dancer2::Core::Role::ConfigReader>
+to manage the Dancer2 configuration.
```

Comment:
I don't think it is advantageous to write the documentation as a "diff" from the previous version. It should just document the way it is.

At most, if you worried people who read this will be confused,d you could say, "starting from version XXXX, the L<ConfigReader> works this way..." or something. However, writing this in a role that is only used by developers is probably not necessary.

It's even more confusing because the class isn't called `Role::ConfigReader` but `Role::Config` and refers to a previously existing `Role::ConfigReader` even though there is a different `Role::ConfigReader` at the end of these commits.

Reply:
Rewritten.